lasagne/regularization.py

```diff
@@ -1,3 +1,53 @@
+"""
+Functions to apply regularization to the weights in a network.
+
+We provide functions to calculate the L1 and L2 penalty. Penalty functions
+take a tensor as input and calculate the penalty contribution from that tensor:
+
+.. autosummary::
+    :nosignatures:
```

Comment:
That would look like:
![signatures](https://cloud.githubusercontent.com/assets/2623134/8455527/a13ddbb6-2005-11e5-9ca2-0beaca52dd6f.png)